microvm-init/init

```diff
@@ -78,6 +78,8 @@ if [ -e /dev/vda ]; then
 
 	# Make /home/build world writeable under /mount
 	chmod 0777 /mount/home/build
+	# Make it accessible also from initramfs
+	mount --bind /mount/home/build /home/build
```

Comment:
I'm worried about the combination of these commands.

When I had `/home/build` more permissively permissioned (in an iteration), sshd was rejecting SSH requests.

Reply:
here it did not give problems to me
I suspect the problem is more on the keys permissions than home itself probably

Reply:
Now I'm avoiding sharing it, and workspacetar will just directly tar from /mount/home/build